oai_agents/common/learner.py

```diff
@@ -0,0 +1,29 @@
+class Learner:
+    magnifier = 3
```

Comment:
It is 3 because right now it only work well with 3. (I also tested with some other parameters.)

I don't understand how to use this parameter at this point. We need to understand more before deciding this use.

Reply:
magnifier is used to enlarge the received reward. After the magnification, the reward may boost the gradient. I need to confirm it after reading more on PPO theory.

Reply:
ok sounds good. Can you put that in a comment in the code so we don't forget about it